.gitlab-ci.yml

```diff
@@ -1,28 +1,23 @@
-image: python:3.7
+# GitLab Dependency Proxy to get rid of Docker hub pull limit issue
+# https://docs.gitlab.com/ee/user/packages/dependency_proxy/#authenticate-within-cicd
+image: ${CI_DEPENDENCY_PROXY_GROUP_IMAGE_PREFIX}/python:3.7
 
 # GitLab CI uses the services keyword to define what docker containers should be linked with our base image.
 # Since we are using docker as our image we'll be using docker:dind i.e. Docker-In-Docker-Image
 services:
-  - name: docker:19.03.12-dind
+  - name: ${CI_DEPENDENCY_PROXY_GROUP_IMAGE_PREFIX}/docker:dind
 
 variables:
-  # Environemt Variable for docker:dind service explaining to use overlay2 as supporting driver for docker
```

Comment:
Why delete this comment?  Is it still relevant for this line:

`DOCKER_DRIVER: overlay2`

Reply:
I'll add it back